application/db/type.go

```diff
@@ -5,7 +5,7 @@
 
 package db
 
-type DBI interface {
+type KeyValue interface {
```

Comment:
why changing name?

Reply:
Prefix `DB` in `DBI` doesn't give any context already having a package name `db`. It's like `db.DBI`